build_projects/update-dependencies/update-dependencies.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+set -e
+
+SOURCE="${BASH_SOURCE[0]}"
+while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symlink
+  DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
+  SOURCE="$(readlink "$SOURCE")"
+  [[ "$SOURCE" != /* ]] && SOURCE="$DIR/$SOURCE" # if $SOURCE was a relative symlink, we need to resolve it relative to the path where the symlink file was located
+done
+DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
+
+REPO_ROOT="$DIR/../.."
+PROJECT_PATH="$DIR/update-dependencies.csproj"
+
+# Some things depend on HOME and it may not be set. We should fix those things, but until then, we just patch a value in
```

Comment:
ok, I'm fine with keeping this if it is needed.  I just didn't know why it was necessary.

---
In reply to: [100941962](https://github.com/dotnet/cli/pull/5695#discussion_r100941962) [](ancestors = 100941962)